# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1523035982 14400
#      Fri Apr 06 13:33:02 2018 -0400
# Branch stable
# Node ID a038cbe3e6e1047115dae4d41f614dfb0c5344da
# Parent  f18f4700aded054afaca50167aba8a272a894e7c
* install.txi: Provide URL for Qt.

diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -307,19 +307,19 @@ provide improved performance for the fun
 
 @item QScintilla
 Source code highlighter and manipulator; a Qt port of Scintilla
 (@url{http://www.riverbankcomputing.co.uk/software/qscintilla}).
 QScintilla is used for syntax highlighting and code completion in the
 GUI.
 
 @item Qt
-GUI and utility libraries (@url{}).  Qt is required for building the GUI.
-It is a large framework, but the only components required are the GUI,
-core, and network modules.
+GUI and utility libraries (@url{https://www.qt.io}).  Qt is required for
+building the GUI.  It is a large framework, but the only components
+required are the GUI, core, and network modules.
 
 @item SuiteSparse
 Sparse matrix factorization library
 (@url{http://www.cise.ufl.edu/research/sparse/SuiteSparse}).
 SuiteSparse is required to provide sparse matrix factorizations and
 solution of linear equations for sparse systems.
 
 @item SUNDIALS
